scripts/kind-e2e/e2e.sh

```diff
@@ -322,16 +331,108 @@ setup_custom_cni
 if [[ $3 = true ]]; then
     enable_logging
 fi
-install_helm
-if [[ $4 = true ]]; then
-    enable_kubefed
+
+if [[ $5 = operator ]]; then
+    operator=true
+    # TODO: Convert these Broker-deploy steps to use Operator instead of Helm
```

Comment:
Seems like they should be outside the if, since you do them in both cases.

Reply:
Good suggestion, thanks, done in the latest version. :+1: